weed/pb/grpc_client_server.go

```diff
@@ -65,15 +66,17 @@ func NewGrpcServer(opts ...grpc.ServerOption) *grpc.Server {
 	return grpc.NewServer(options...)
 }
 
-func GrpcDial(ctx context.Context, address string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
+func GrpcDial(ctx context.Context, address string, waitForReady bool, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
 	// opts = append(opts, grpc.WithBlock())
 	// opts = append(opts, grpc.WithTimeout(time.Duration(5*time.Second)))
 	var options []grpc.DialOption
+
 	options = append(options,
 		// grpc.WithTransportCredentials(insecure.NewCredentials()),
 		grpc.WithDefaultCallOptions(
 			grpc.MaxCallSendMsgSize(Max_Message_Size),
 			grpc.MaxCallRecvMsgSize(Max_Message_Size),
+			grpc.WaitForReady(waitForReady),
```

Comment:
how about we just use wait for ready for all the grpc calls?

Reply:
I don't understand how it's gonna be helpful. If some server goes down and we try to wait for connection with it without timeout, then goroutine that made this call will be blocked. Why do you think it's appropriate?
I work on this project for a short time and don't know all specifics, but will WaitForReady on every grpc call really work?

Reply:
> how about we just use wait for ready for all the grpc calls?
We already tried it. :) 
https://github.com/seaweedfs/seaweedfs/pull/1577/files

Reply:
I don't understand what has changed since https://github.com/seaweedfs/seaweedfs/issues/1820 where this option was removed. You still insist on enabling WaitForReady for all grpc calls?